blog/june-2019-workshop.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html
+  PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-105598606-2"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'UA-105598606-2');
+  </script>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta http-equiv="Content-Style-Type" content="text/css" />
+  <meta name="generator" content="pandoc" />
+  <meta name="author" content="Henry Ehrenberg, Alex Ratner, Paroma Varma, Braden Hancock, Vincent Chen, Chris Ré" />
+  <title>Recap of June's Snorkel Workshop</title>
+  <link rel="stylesheet" href="blog.css">
+  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML-full"
+    type="text/javascript"></script>
+</head>
+
+<body>
+  <div id="header">
+
+    <!-- Title -->
+    <h1 class="title">
+      Recap of June's Snorkel Workshop
+    </h1>
+    <a href='https://github.com/henryre/'>Henry Ehrenberg</a>, <a href='https://ajratner.github.io'>Alex Ratner</a>, <a
+      href='https://www.paroma.xyz/'>Paroma Varma</a>, <a href='https://www.bradenhancock.com'>Braden Hancock</a>, <a
+      href='https://vincentsc.com'>Vincent S. Chen</a>, <a href='https://cs.stanford.edu/people/chrismre/'>Chris Ré</a>,
+    <em>and referencing work by &nbsp;<a href='https://cs.stanford.edu/people/chrismre/#students'>many
+        other&nbsp;members&nbsp;of&nbsp;Hazy&nbsp;Lab</em></a></br>
+    <br>
+    <a href="http://snorkel.stanford.edu">Back to the Snorkel Blog</a>
+  </div>
+
+  <br>
+
+  <div class="figure">
+    <img src="june-2019-workshop-figs/workshop.png" width="50%" class="centered" />
+  </div>
+
+  <br>
+
+  On June 24th and 25th, we hosted a Snorkel workshop at Stanford to talk about
+  recent research around Snorkel, discuss applications and extensions, and most
+  importantly from our end, get feedback on the upcoming v0.9 release of Snorkel.
+  This upcoming release is our most significant upgrade yet,
+  integrating research work from the last two years
+  (<a href="https://arxiv.org/abs/1711.10160">VLDBJ 2019</a>,
+  <a href="https://arxiv.org/abs/1903.05844">ICML 2019</a>,
+  <a href="https://arxiv.org/abs/1812.00417">SIGMOD 2019</a>,
+  <a href="https://arxiv.org/abs/1810.02840">AAAI 2019</a>,
+  <a href="https://ajratner.github.io/assets/papers/deem-metal-prototype.pdf">DEEM 2018</a>,
+  <a href="https://arxiv.org/abs/1709.01643">NeurIPS 2017</a>),
+  new operators for building and managing training data, the full code to reproduce our
```

Comment:
"scripts demonstrating how to reproduce our recent state-of-the-art results on the public benchmark SuperGLUE[link]"